src/sort-files.js

```diff
@@ -52,8 +52,26 @@ module.exports = function sortFiles(packageJson) {
 
   const isPackageMain = (filepath) => filepath === main;
   const ignored = or(ALWAYS_INCLUDED, ALWAYS_EXCLUDED, isPackageMain);
+  const directoriesFirst = (a, b) => {
+    if (a.endsWith('/') && b.endsWith('/')) {
+      return 0;
+    } else if (a.endsWith('/')) {
+      return -1;
+    } else {
+      return 1;
+    }
+  };
```

Comment:
I'm pretty sure this can be done in a less verbose way, if anyone has suggestions, please comment, and I'll update! :) :+1:

Reply:
💭 You could make use of [sort-order](https://github.com/cameronhunter/sort-order) :)

Reply:
Sounds good :+1: How exactly would that look like? :)